Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        prom/busybox:glibc
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9100
+COPY node_exporter  /bin/node_exporter
+
+EXPOSE     9100
+ENTRYPOINT [ "/bin/node_exporter" ]
```

Comment:
Once more the discussion, but I'm not a fan of default entrypoints in Dockerfile. It hides the executable being executed from the user and makes it more complicated to run a different command. I vote to leave it as it was before.